components/definition/cellRules/NetPyNECellRule.js

```diff
@@ -63,17 +82,32 @@ export default class NetPyNECellRule extends React.Component {
       />
 
       <br/>
-
-      <NetPyNEField id="netParams.cellParams.conds.cellModel" >
-        <PythonControlledTextField 
-        	model={"netParams.cellParams['" + this.props.name + "']['conds']['cellModel']"} 
-        	id="cellRuleCellModel"/>
+      
+      <NetPyNEField id={"netParams.cellParams.conds.pop"} >
+        <PythonMethodControlledSelectField
+          model={"netParams.cellParams['" + this.props.name + "']['conds']['pop']"}
```

Comment:
Not sure if this should be `this.props.name` or `this.state.currentName`. I would say the second but it is true there is some code in other files (for instance NetPyNEStimulationTarget) that uses props.name. Can you please double check this?